pkg/controller/servicebindingrequest/binder.go

```diff
@@ -239,15 +253,15 @@ func (b *Binder) update(objList *ustrv1.UnstructuredList) ([]string, error) {
 			}
 		}
 
-		logger.Info("Updating object...")
+		logger.Info("Updating object in Kube...")
 		if err := b.client.Update(b.ctx, updatedObj); err != nil {
 			return nil, err
 		}
 
 		// recording object as updated
 		updatedObjectNames = append(updatedObjectNames, name)
+		logger.Info(updatedObjectNames[0])
```

Comment:
If there are no updated objects, `updatedObjectNames` would be empty in that case. In that case here we are accessing the first element from the slice which would cause panic. 
cc: @sbose78

Reply:
True!

Reply:
Removed it :) Was pretty unnecessary.